bowconvert_test.go

```diff
@@ -124,3 +126,157 @@ func TestToString(t *testing.T) {
 	require.True(t, ok)
 	assert.Equal(t, "0", v)
 }
+
+func TestToTimestamp(t *testing.T) {
+	t.Run("Sec", func(t *testing.T) {
+		v, ok := ToTimestampSec(true)
+		require.True(t, ok)
+		assert.Equal(t, arrow.Timestamp(1), v)
+
+		v, ok = ToTimestampSec(false)
+		require.True(t, ok)
+		assert.Equal(t, arrow.Timestamp(0), v)
+
+		v, ok = ToTimestampSec(0.)
+		require.True(t, ok)
+		assert.Equal(t, arrow.Timestamp(0), v)
+
+		v, ok = ToTimestampSec(0)
+		require.True(t, ok)
+		assert.Equal(t, arrow.Timestamp(0), v)
+
+		v, ok = ToTimestampSec("0")
+		require.True(t, ok)
+		assert.Equal(t, arrow.Timestamp(0), v)
+		assert.Equal(t, "1970-01-01T00:00:00Z",
+			v.ToTime(arrow.Second).Format(time.RFC3339Nano))
+
+		formattedTimeSec := "2022-04-27T00:00:00Z"
+		ti, err := time.Parse(time.RFC3339, formattedTimeSec)
+		require.NoError(t, err)
+
+		v, ok = ToTimestampSec(ti.Unix())
+		require.True(t, ok)
+		assert.Equal(t, "2022-04-27T00:00:00Z",
+			v.ToTime(arrow.Second).Format(time.RFC3339Nano))
+
+		v, ok = ToTimestampSec(formattedTimeSec)
+		require.True(t, ok)
+		assert.Equal(t, "2022-04-27T00:00:00Z",
+			v.ToTime(arrow.Second).Format(time.RFC3339Nano))
+	})
+
+	t.Run("Milli", func(t *testing.T) {
+		v, ok := ToTimestampMilli(true)
```

Comment:
I'd split every three-lines assertion to have tests more unitary and not make an entire test fail when only a specific behaviour is broken

Reply:
I get the point, but it would add 110 lines of code in this file and make the code much harder to read. Do you think this is worth it?

Reply:
Maybe that's another problem of having such big files 🙈
Let's skip it